markdown/boomi-v2-getting-started/boomi-v2-getting-started.md

```diff
@@ -161,54 +158,37 @@ Select your newly created environment and search for your atom to attach it
 Positive
 : Note: a blue icon next to your newly created Atom means that it has an Online Status
 
-#### 5. Install the Solace Connector
-
-Click on the Settings > Account information for your account name
-![account_setup](img/account_setup.png "account_setup")
-
-In the left hand column, click on Publisher
-![publisher](img/publisher.png "publisher")
-Fill out the required information, making sure to click on the box for "Enable connector SDK development" Click on Save.
-
-Now, click on Developer at the bottom of the left hand column.
-![developer](img/developer.png "developer")
+#### 🎉 Viola! Now you have your Solace and Boomi environments setup
```

Comment:
Line 99: I believe the Solace connector doesn't quite work on Atom Cloud. I don't know the specifics but you might want to check with Mike Hilmen or Andrew MacKenzie.

Reply:
Good point I will check and confirm

Reply:
I just checked and confirmed it works 👍 I ran a quick test as well

Reply:
cool